src/smartthings/smartapps/smartthings/amazon-echo-home-skill.src/amazon-echo-home-skill.groovy

```diff
@@ -15,12 +15,23 @@ definition(
 		iconX2Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/App-AmazonEcho@2x.png",
 		iconX3Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/App-AmazonEcho@3x.png",
 		oauth: [displayName: "Amazon Echo (QA)", displayLink: ""]
+		singleInstance: true
```

Comment:
@larsfinander  You need a comma at the end of the preceding line ;)
Pending that, I approve ( we need a 99 44/100 emoji )